qrcode_generator_random.py

```diff
@@ -0,0 +1,30 @@
+import qrcode
+import random
+
+def linkGen(n):
+
+    random_nums = []
+    while len(random_nums) != n:
```

Comment:
nit: if n > (99999999 - 10000000 = 89999999) then the loop will run indefinitely.